src/Post/Post.js

```diff
@@ -174,49 +245,110 @@ function CommentSection({ index, comments, isExpanded, setIsExpanded, datum }) {
         >
           <Avatar src={datum.user_thumbnail} alt={datum.user_name} />
           <p className="post-caption-text">
-            <strong>{datum?.user_name}</strong> {datum?.post_caption}
+            <span className="comment-username">{datum?.user_name}</span>{' '}
+            {datum?.post_caption}
           </p>
         </section>
       )}
-      {commentsSummary.map((comment) => (
-        <div
-          className={classnames('comment-wrapper', {
-            'is-expanded': isExpanded,
-          })}
-        >
-          {isExpanded && <Avatar src={comment.user_thumbnail} size="32" />}
-          <div className="comment-body">
-            <p className="comment-text">
-              <strong>{comment.user_name}</strong> {comment.comment}
-            </p>
-            {isExpanded && (
-              <div className="comment-action-container">
-                <time className="comment-action">2 w</time>
-                <button className="comment-action">
-                  <strong>
-                    {comment.comment_likes}
-                    {comment.comment_likes > 0 ? ' likes' : ' like'}
-                  </strong>
-                </button>
-                <button className="comment-action">
-                  <strong>Reply</strong>
-                </button>
-              </div>
-            )}
-          </div>
-          <svg
-            aria-label="Like"
-            className="action-icon"
-            color={comment.is_liked_by_user ? '#ed4956' : '#262626'}
-            height="12"
-            role="img"
-            viewBox="0 0 48 48"
-            width="12"
+      {commentsSummary.map((comment, index) => {
+        return (
+          <div
+            key={comment.comment_id}
+            className={classnames('comment-wrapper', {
+              'is-expanded': isExpanded,
+            })}
+            onMouseEnter={() =>
+              setOnCommentHover(
+                onCommentHover === comment.comment_id
+                  ? undefined
+                  : comment.comment_id
+              )
+            }
+            onMouseLeave={() => setOnCommentHover(undefined)}
```

Comment:
Oh great work on this! But you can solve it using CSS only, which is faster and cleaner.

All you need to do is 

```css
.three-dots-button {
	visibility: hidden
}
.comment-action-container:hover .three-dots-button {
	visibility: visible
}
````